docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md

```diff
@@ -93,7 +102,7 @@ To enable LDAP authentication for YugabyteDB Anywhere login, you can perform a n
   --data-raw '[LDAP DN]'
   ```
 
-  <br>Replace `[LDAP DN]` with the actual value, as per the following example: <br>
+  Replace `[LDAP DN]` with the actual value, as per the following example: <br>
 
   `,DC=yugabyte,DC=com`
```

Comment:
Can you just remove the `,` before `DC=yugabyte,DC=com`? We don't need that anymore. Thanks!

Reply:
Fixed